www/data_service/src/md_demo/demo.controller.coffee

```diff
@@ -0,0 +1,10 @@
+class Demo extends Controller('app')
+    constructor: ($scope, $timeout, $log, dataService) ->
+        opened = dataService.open()
+        opened.closeOnDestroy($scope)
+        opened.getBuilders(name: 'www').then (@builders) =>
+            @builders[0].loadBuildrequests()
+
+        @unsub = ->
```

Comment:
Why do you define this method in the constructor other than define it a a class method?